src/index.html

```diff
@@ -11,8 +11,53 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Nunito:ital,wght@0,200..1000;1,200..1000&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <div class="card__content">
+        <div class="card__item-image"></div>
+        <div class="card__item">
+          <div class="card__item-name">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </div>
+          <div class="card__item-code">Product code: 195434</div>
```

Comment:
use semantic tags for text. Please make it for everywhere.
```suggestion
          <h3 class="card__item-name">
            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
          </h3>
          <pclass="card__item-code">Product code: 195434</p>
```